app/person/controllers.js

```diff
@@ -3,52 +3,51 @@ const axios = require('axios')
 const presenters = require('../../common/presenters')
 const nunjucksGlobals = require('../../config/nunjucks/globals')
 
-module.exports = {
-  image: fallbackImage => {
-    return async (req, res) => {
-      try {
-        const imageUrl = await req.services.person.getImageUrl(
-          req.params.personId
-        )
-        const response = await axios.get(imageUrl, {
-          responseType: 'arraybuffer',
-        })
+function image(fallbackImage) {
```

Comment:
Wonder if its actually worth extracting this one layer further into separate files like some of the other larger apps if we know there will be another one (possibly two) controllers?

So something like:

```
/controllers
  image.js
  image.test.js
  personal-details.js
  ...
```

Reply:
Yeah, I wondered about this too. I decided to leave it because the controllers and middleware were quite small, but since you've raised it, I'll add a commit which does this!